.github/workflows/ci-build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4]
+        php: [7.2, 7.3, 7.4, 8.0]
```

Comment:
@wolfy-j What about 7.2? I assume that we should remove PHP 7.2 support, shouldn't we?

Reply:
Yeah, we can drop it from tests.

Reply:
@VitalyArt Could you please remove PHP 7.2 everywhere from `CI`, `bors.toml` and `composer.json`?

Reply:
@48d90782 done